app/(home)/github/page.tsx

```
@@ -0,0 +1,302 @@
+        <div className="mx-auto flex max-w-xl flex-col py-12 sm:py-20">
+          <div className="mb-8">
+            <RepositoryBadges />
+          </div>
+
+          <div className="divide-y divide-zinc-200 dark:divide-zinc-800">
+            {releases.length > 0 ? (
+              releases.map((release) => (
+                <article key={release.id} className="flex flex-col py-16">
+                  <div className="mb-4 flex w-full flex-wrap items-center justify-between gap-x-4 gap-y-2 text-xs">
+                    <div className="flex items-center gap-3">
+                      <BadgeButton
```

ESLint

[error] 'BadgeButton' is not defined.

(react/jsx-no-undef)